content/800-data-platform/06-query-console.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Query Console'
+metaTitle: 'Query Console'
+metaDescription: 'Query Console'
+toc: true
+tocDepth: 3
+---
+
+<TopBlock>
+
+Use the Query Console to run the 3 pre-defined `findMany()` test queries or add and test queries of your own using the [Prisma Client API for model queries](/reference/api-reference/prisma-client-reference#model-queries).
+
+</TopBlock>
+
+## UI components
+
+The Query Console is comprised of 2 major UI components - **Query Editor** and **Query Results Console**.
+
+<img src="https://res.cloudinary.com/prismaio/image/upload/v1657148399/docs/pdp/pdp-query-console-components_w98dp5.png" alt="Query Console - UI Components"/>
+
+| UI component          | Description                                                                         |
+| --------------------- | ----------------------------------------------------------------------------------- |
+| Query Editor          | • Add, edit, delete queries using the Prisma Client API<br />• Run a selected query |
+| Query Results Console | View the results from the selected query                                            |
+
+## Default model queries
+
+- `findMany()` for the first 100 records in the `User` model
+- `findMany()` for the first 100 records in the `User` model that match a criteria
+- `findMany()` for the first 100 records in the `User` model but only return data from 2 fields
+
+## Run a query
+
+You can test and run queries for your app using Prisma Client from the Query Console and view their results on the right side of the screen.
+
+**Steps**
+
+1. In the Query Editor, place the cursor anywhere in a query block.
+2. Run the query using one of the 3 available method.
+   - Click the **Run active query** button in the upper-left.
+   - Click the **Play** (**Run query**) button at the left margin of the editor.
+   - Press <kbd>Cmd</kbd> + <kbd>Enter</kbd> (on macOS) or <kbd>Ctrl</kbd> + <kbd>Enter</kbd> (on Windows).
+   <img src="https://res.cloudinary.com/prismaio/image/upload/v1657148832/docs/pdp/pdp-query-console-run-query_lgc5nb.png" width="450px" alt="Query Console - Run a query"/>
+
+**Result**
+
+The query results appeare in the right side in the Query Results Console.
+
+## Add a query
+
+The Query Console supports the autocompletion you get when you use your Prisma Client API in Visual Studio Code.
```

Comment:
```suggestion
The Query Console supports the same autocompletion you get when you use your Prisma Client API in Visual Studio Code or other modern IDEs.
```

Reply:
Changed as you suggested. Also added 'syntax highlighting' as that is also available.